app/controllers/users/email_confirmations_controller.rb

```diff
@@ -44,10 +44,14 @@ def process_successful_confirmation(email_address)
       confirm_and_notify(email_address)
       if current_user
         flash[:success] = t('devise.confirmations.confirmed')
-        redirect_to account_url
+        if params[:request_id]
```

Comment:
I'm wondering if we should do something like [what's done in the `SessionsController`](https://github.com/18F/identity-idp/blob/bed2a55da0f156d7226f1e871f7c9ee4dcbe54ac/app/controllers/users/sessions_controller.rb#L243-L246) to persist the `request_id` in the session. This _might_ work, but it's probably not respecting the `request_id` from parameters.

Reply:
I wasn't sure session would persist long enough